omniledger/service/service.go

```diff
@@ -172,6 +210,7 @@ func (s *Service) createNewBlock(scID skipchain.SkipBlockID, r *onet.Roster, ts
 		sb.MaximumHeight = 10
 		sb.BaseHeight = 10
 		// We have to register the verification functions in the genesis block
+		// TODO why are we not doing tryHash here?
```

Comment:
Because this is a throw-away collection specifically created for the genesis block. The real collection will be created later, once the block is pushed to the other nodes.